spec/helpers/application_helper_spec.rb

```diff
@@ -18,4 +18,10 @@
       expect(helper.formatted_month_name(1)).to eq('janeiro')
     end
   end
+
+  describe 'short month name' do
```

Comment:
**suggestion:** I normally prefer to describe method tests with the method name. In this case:

```suggestion
  describe '#formatted_month_short' do
```